plugins/dspam/dspam_

```diff
@@ -211,7 +215,15 @@ file_is_locked() {
 	local file=$1
 	local lock=$1.lock
 
-	[ -f $lock ] && debug file $file is locked && return 0 # EX_OK
+	if [ -f $lock ];then
```

Comment:
Please quote variables, whenever it is possible.
Probably `shellcheck -s dash dspam_` would help you finding all occurrences.